pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -402,12 +401,14 @@ func modifyOutboundRouteConfig(push *model.PushContext, in *plugin.InputParams,
 					DisableCheckCalls: disablePolicyChecks(outbound, in.Env.Mesh, in.Node),
 					MixerAttributes:   &mpb.Attributes{Attributes: attrs},
 					ForwardAttributes: &mpb.Attributes{Attributes: attrs},
+					QuotaSpec:         getQuotaSpec(in, hostname),
```

Comment:
Quota checks are subject to policy check condition. Outbound policy checks are rarely applied. Is this intentional to rely on outbound policy checks?

Reply:
I see, so you assume that egress gateway is used, and we do apply outbound checks on gateways. 
I think it's worth guarding quotaspecs by disable check calls in these protos, in the interest of saving space.

Reply:
thanks for the feedback.. updating..